addon/components/frost-checkbox.js

```diff
@@ -1,34 +1,50 @@
+/**
+ * Component definition for the frost-checkbox component
+ */
```

Comment:
This seems redundant - I think we need to make some basic assumptions that people know how Ember projects are structured